integrations/gitlab_v1/config.yaml

```diff
@@ -0,0 +1,24 @@
+initializePortResources: true
+scheduledResyncInterval: 1440  # 60 minutes X 24 hours = 1 day
+port:
+  clientId: "{{ from env PORT_CLIENT_ID }}"  # Can be loaded using environment variable: PORT_CLIENT_ID
+  clientSecret: "{{ from env PORT_CLIENT_SECRET }}"  # Can be loaded using environment variable: PORT_CLIENT_SECRET
+
+
+# The event listener to use for the integration service.
+eventListener:
+  type: POLLING
+
+
+integration:
+  # The identifier of this integration instance.
+  identifier: "gitlab_v1"
+  # The type of the integration.
+  type: "Gitlab"
+  config:
+    gitlabToken:
+      - "{{ from env GITLAB_TOKEN }}"
+      - "{{ from env GITLAB_TOKEN2 }}"
+    gitlabUrl: "https://gitlab.com"
+    webhookUrl: "{{ from env WEBHOOK_URL }}"
+    webhookSecret: "{{ from env WEBHOOK_SECRET }}"
```

Comment:
we don't use the config.yaml file anymore. so delete it

Reply:
What would be the alternative to this, 
This is what I was using for reference . 
[Develop an Integration](https://github.com/port-labs/ocean/blob/main/docs/framework-guides/docs/develop-an-integration/develop-an-integration.md)